python/ray/serve/tests/test_controller_recovery.py

```diff
@@ -200,5 +200,48 @@ def make_nonblocking_calls(expected, expect_blocking=False, num_returns=1):
     make_nonblocking_calls({"2": 2}, num_returns=2)
 
 
+def test_controller_recover_initializing_actor(serve_instance):
+    """Recover the actor which is under PENDING_INITIALIZATION"""
+
+    signal = SignalActor.remote()
+    signal2 = SignalActor.remote()
+    client = serve_instance
+
+    @ray.remote
+    def pending_init_indicator():
+        ray.get(signal2.wait.remote())
+        return True
+
+    @serve.deployment
+    class V1:
+        def __init__(self):
+            signal2.send.remote()
+            ray.get(signal.wait.remote())
```

Comment:
```suggestion
            await signal.wait.remote()
```

Reply:
I think it make no difference for this test case.

Reply:
Is the controller able to call the replica during recovery while the replica is stuck on `ray.get()`, since its a blocking call?